app/services/licences/view-licence-returns.service.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+/**
+ * Orchestrates fetching and presenting the data needed for the licence summary page
+ * @module ViewLicenceSummaryService
+ */
+
+const FetchLicenceReturnsService = require('./fetch-licence-returns.service')
+const ViewLicenceService = require('./view-licence.service')
+const ViewLicenceReturnsPresenter = require('../../presenters/licences/view-licence-returns.presenter')
+const PaginatorPresenter = require('../../presenters/paginator.presenter')
+/**
```

Comment:
alan is going to point out a NIT here of an extra empty line between the declarations and the comments